docker/Dockerfile

```diff
@@ -26,9 +27,22 @@ RUN pip3 install datamodel_code_generator
 
 WORKDIR /sgl-workspace
 
-ARG CUDA_VERSION
+FROM base AS sglang-remote
+RUN git clone ${SGL_REPO} \
```

Comment:
Does it mean we need to explicitly pass the git URL as an env var?

Reply:
Good catch, in another PR (#3202) I have the missing part of these changes.
Will port them over now.

Reply:
Thanks